### ISSUE TITLE
Could canvasXpress respect positional scale limits and expand arguments?

### ISSUE THREAD

**Issue by davidhodge931** [neuhausi/canvasXpress#153]
```
library(tidyverse)
library(palmerpenguins)
library(ggblanket)

p <- penguins |> 
  group_by(species) |> 
  summarise(body_mass_g = mean(body_mass_g, na.rm = TRUE)) |> 
  ggplot(aes(x = species, y = body_mass_g)) +
  geom_point() +
  scale_y_continuous(limits = c(0, 7000), 
                     breaks = scales::breaks_width(500),
                     expand = c(0, 0)) 

p

canvasXpress::canvasXpress(p)
```

**Comment by neuhausi** [comment]
Fix in version 48.9
![cX-htmlwidget-a292c0c1f49c1079bd04-cx (1)](https://github.com/neuhausi/canvasXpress/assets/254923/5d91e59f-80cd-43b9-876e-b7ceb035525b)

**Comment by davidhodge931** [comment]
It is respecting limits now, but still not expand
